index.html

```diff
@@ -0,0 +1,229 @@
+      <div class="button_container">
+        <button type="button" class="button" onclick="alert('Contact redirection')">Contact me!</button>
+        <button type="button" class="button" onclick="alert('CV redirection')">Check my CV</button>
+      </div>
+    </section>
+
+    <section class="about_me" id="about_me">
+      <h2>About me</h2>
+      <div class="photo_personalData_description">
+        <img src="./img/avatar.png" alt="Photo of me" width="96px" height="96px">
+        <div class="personalData">
+          <ul>
+            <li>Full name: <b class="about_me_distinctive_text">Konrad Kalita</b></li>
+            <li>Age: <b class="about_me_distinctive_text">20 years</b></li>
+            <li>Profile: <b class="about_me_distinctive_text">Fullstack</b></li>
+            <li>Telephone: <b class="about_me_distinctive_text">+48 123 456 789</b></li>
+            <li>E-mail: <b class="about_me_distinctive_text">xyz@gmail.com</b></li>
+          </ul>
+        </div>
+        <div class="description">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin
+          consectetur blandit massa, vitae condimentum justo viverra at. Suspendisse 
+          id enim congue, iaculis eros a, volutpat ex. Interdum et malesuada fames ac
+          ante ipsum primis in faucibus. Morbi arcu dolor, porta eget enim volutpat, 
+          ullamcorper venenatis orci. Phasellus rhoncus auctor nisi. Aliquam erat vol
+          utpat. Fusce facilisis libero quam, vel venenatis ipsum tin
+          cidunt sed. Sed congue lorem nec eros pellentesque ultrices. Pellentesque at
+          laoreet arcu, a suscipit ligula. Aliquam volutpat tortor condimentum consec
+          tetur tempor. Cras ullamcorper lectus nec odio consequat interdum
+        </div>
+      </div>
+    </section>
+
+    <section class="skills" id="skills">
+      <h2>Skills</h2>
+      <div>
+        <ul class="core_skills">
+          <li class="certain_skill">
+            <img src="./img/skillsIcons/icon_UxUi.png" class="certain_skill_photo" alt="UX/UI Figma" width="83px" height="83px">
+            <b class="distinctive_text">UX/UI</b>
+            Figma
+          </li>
+          <li class="certain_skill has_special_borders">
+            <img src="./img/skillsIcons/icon_frontend.png" class="certain_skill_photo" alt="HTML CSS JAVASCRIPT" width="83px" height="83px">
+            <b class="distinctive_text">Front End</b>
+            Html, CSS, Javascript
+          </li>
+          <li class="certain_skill">
+            <img src="./img/skillsIcons/icon_backend.png" class="certain_skill_photo" alt="PHP, Node.js" width="83px" height="83px">
+            <b class="distinctive_text">Back End</b>
+            PHP, Node.js
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="portfolio" id="portfolio">
+      <h2>Portfolio</h2>
+      <div class="design_projects">
+        <div class="item-a">
```

Comment:
Nazwy `item-a item-b item-c` mało nam mówią. Oraz mają bardzo wspólnych styli. Może dobrze by było to sprecyzować i wydzielić rzeczy wspólne do osobnej klasy? Naprzykład do klasy `item` tj:
```
display: flex;
  background-repeat: no-repeat;
  background-size: cover;
  transition: transform 0.4s ease-in-out;
```
A następnie do każdej dodać już bardzo indywidualną klase tj np: `item-first`